tests/unit/test_inference.py

```diff
@@ -17,7 +17,7 @@
     from lm_eval.evaluator import evaluate
     from transformers import pipeline, AutoModelForCausalLM, AutoTokenizer
     from huggingface_hub import HfApi
-except ImportError:
+except (ImportError, AssertionError) as e:
```

Comment:
Is it safe to skip every assertion error?

Reply:
That's a good point, actually this setup could skip all tests if there is an AssertionError and ImportError with `-m "inference"` and show passing. I think we can add another check to see if the `inference` tag was passed to pytest (otherwise, we don't care if there is an error importing here)